frontend/src/metabase/visualizations/echarts/tooltip/index.tsx

```diff
@@ -130,28 +130,29 @@ export const useInjectSeriesColorsClasses = (hexColors: string[]) => {
       .join("\n");
   }, [hexColors]);
 
-  const style = useMemo(
+  return useMemo(
     () =>
       cssString !== null ? (
         <style nonce={window.MetabaseNonce}>{cssString}</style>
       ) : null,
     [cssString],
   );
-
-  return style;
 };
 
 export const useClickedStateTooltipSync = (
   chart?: EChartsType,
   clicked?: ClickObject | null,
 ) => {
-  useEffect(
-    function toggleTooltip() {
-      const isTooltipEnabled = clicked == null;
-      chart?.setOption({ tooltip: { show: isTooltipEnabled } }, false);
-    },
-    [chart, clicked],
-  );
+  useEffect(() => {
```

Comment:
The tooltip should become disabled to prevent it from appearing when the click popover is visible:
<img width="453" height="504" alt="Screenshot 2025-08-22 at 4 53 14 PM" src="https://github.com/user-attachments/assets/4c6daefd-b4e1-41ee-89ac-12c8240a585a" />